### PR TITLE
Feature/pelagos 3984 remove mentions of gomri

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: templates/DIF/difForm.html.twig [hunk 1]
```diff
@@ -229,13 +229,13 @@
...
                 <div class="tooltiptext">
-                    <p>The approximate time period when the data is expected to be collected and generated based on the information provided in your GoMRI proposal.</p>
+                    <p>The approximate time period when the data is expected to be collected and generated based on the information provided in your proposal.</p>
                 </div>
             </span>
             <label class="emRequired"><b>Estimated Data Sampling/Generation Period: </b></label>
             <div align="center">
                 <label for="estimatedStartDate" class="error"></label>
-                <label for="estimatedEndDate" class="error"/></label>
+                <label for="estimatedEndDate" class="error"></label>
```
Comment (fightingtexasaggie) [inline templates/DIF/difForm.html.twig:238]: good catch

File: templates/DIF/email/data-managers/data-managers.dif-submitted.email.twig [hunk 2]
```diff
@@ -3,23 +3,23 @@
...
 
-A Dataset Information Form (DIF) has been saved and submitted to the Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) by {{ user.firstName }} {{ user.lastName }} for dataset {{ dif.dataset.udi }}. {{ user.firstName }} {{ user.lastName }} is a member of {{ user.getResearchGroupNames() | join(', ') }} and the DIF submitted is associated with {{ dif.researchGroup.name }}.
+A Dataset Information Form (DIF) has been saved and submitted to the GRIIDC by {{ user.firstName }} {{ user.lastName }} for dataset {{ dif.dataset.udi }}. {{ user.firstName }} {{ user.lastName }} is a member of {{ user.getResearchGroupNames() | join(', ') }} and the DIF submitted is associated with {{ dif.researchGroup.name }}.
 
 Thank you,
 
 The GRIIDC Team
 
-*You are receiving this email because you are identified as a data manager for a Gulf of Mexico Research Initiative Research Consortium. If you believe you should not be receiving this email please notify griidc@gomri.org.*
+*You are receiving this email because you are identified as a data manager for this research group. If you believe you should not be receiving this email please notify griidc@gomri.org.*
```
Comment (fightingtexasaggie) [inline templates/DIF/email/data-managers/data-managers.dif-submitted.email.twig:24]: should we try and mention the RG? This may be a bit too generic.

Comment (praneethpr) [reply templates/DIF/email/data-managers/data-managers.dif-submitted.email.twig:24]: Rosalie gave me the wording

File: templates/Dataland/datasetCitation.html.twig [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{{ dataset.Authors }} ({{ dataset.AcceptedDate|date("Y") }}) {{ dataset.title }}. Distributed by: Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC), Harte Research Institute, Texas A&M University-Corpus Christi. 
+{{ dataset.Authors }} ({{ dataset.AcceptedDate|date("Y") }}) {{ dataset.title }}. Distributed by: GRIIDC, Harte Research Institute, Texas A&M University-Corpus Christi.
```
Comment (fightingtexasaggie) [inline templates/Dataland/datasetCitation.html.twig:1]: Do we need to escape the & character in this context? &amp;

File: templates/DIF/email/data-managers/data-managers.dif-unlocked.email.twig [hunk 4]
```diff
@@ -3,23 +3,23 @@
...
 <br>
-*You are receiving this email because you are identified as a data manager for a Gulf of Mexico Research Initiative Research Consortium. If you believe you should not be receiving this email please notify <a href="mailto:griidc@gomri.org">griidc@gomri.org</a>.*
+*You are receiving this email because you are identified as a data manager for this research group. If you believe you should not be receiving this email please notify <a href="mailto:griidc@gomri.org">griidc@gomri.org</a>.*
 {% endblock %}
 
 {% block body_text %}
 Dear {{ recipient.firstName }} {{ recipient.lastName }},
 
-The Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) has approved a request to unlock the Dataset Information Form (DIF) for dataset {{ dif.dataset.udi }} submitted by {{ user.firstName }} {{ user.lastName }}.
+The GRIIDC has approved a request to unlock the Dataset Information Form (DIF) for dataset {{ dif.dataset.udi }} submitted by {{ user.firstName }} {{ user.lastName }}.
```
Comment (fightingtexasaggie) [inline templates/DIF/email/data-managers/data-managers.dif-unlocked.email.twig:18]: Remove 'The'

**Comment by mickel1138** [comment]
> The only question we should probably ask the PO is whether to start a sentence referring to "The GRIIDC" or should we start with "GRIIDC" instead. This is super minor though, and maybe not worth the worry.

Lets assume all the text is as intended, it was written with PO guidance. If there are any changes, the PO's will let us know in the PO review.

File: templates/DIF/email/data-managers/data-managers.dif-unlocked.email.twig (hunk 4 above)
Comment (praneethpr) [reply templates/DIF/email/data-managers/data-managers.dif-unlocked.email.twig:18]: I'll ask the PO about this

File: templates/Dataland/datasetCitation.html.twig (hunk 3 above)
Comment (praneethpr) [reply templates/Dataland/datasetCitation.html.twig:1]: Thats how it is on prod, do we want t o change it?

Comment (mickel1138) [reply templates/Dataland/datasetCitation.html.twig:1]: Leave it alone I say.

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Pelagos [![Build Status](https://api.travis-ci.com/griidc/pelagos.svg)](https://travis-ci.com/griidc/pelagos)
 
 Pelagos is a system for maintaining a repository of scientific research data.
-Developed and maintained by the The Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC).
+Developed and maintained by the The GRIIDC.
```
Comment (praneethpr) [inline README.md:4]: Remove double "the"

File: templates/Dataland/datasetCitation.html.twig (hunk 3 above)
Comment (fightingtexasaggie) [reply templates/Dataland/datasetCitation.html.twig:1]: ok, if it works there all is well.

**Pull request merged**